server/repositories/cmsQueries/__tests__/HomepageQuery.spec.js

```diff
@@ -0,0 +1,109 @@
+const { HomepageQuery } = require('../homepageQuery');
+
+describe('Homepage query', () => {
+  const query = new HomepageQuery('berwyn', 'prisoner', 1);
+  describe('path', () => {
+    it('should create correct path', async () => {
+      expect(query.path()).toStrictEqual(
+        '/jsonapi/prison/berwyn/node/featured_articles?include=field_moj_featured_tile_large.field_moj_thumbnail_image%2Cfield_moj_featured_tile_large.field_image%2Cfield_moj_featured_tile_small.field_moj_thumbnail_image%2Cfield_moj_featured_tile_small.field_image&page%5Blimit%5D=1&fields%5Bnode--featured_articles%5D=title%2Cdrupal_internal__nid%2Cfield_moj_featured_tile_large%2Cfield_moj_featured_tile_small&fields%5Bnode--page%5D=drupal_internal__nid%2Cfield_moj_thumbnail_image%2Cfield_image%2Ctitle%2Cfield_moj_description%2Cfield_moj_series&fields%5Bnode--moj_video_item%5D=drupal_internal__nid%2Cfield_moj_thumbnail_image%2Cfield_image%2Ctitle%2Cfield_moj_description%2Cfield_moj_series&fields%5Bnode--moj_radio_item%5D=drupal_internal__nid%2Cfield_moj_thumbnail_image%2Cfield_image%2Ctitle%2Cfield_moj_description%2Cfield_moj_series&fields%5Bnode--moj_pdf_item%5D=drupal_internal__nid%2Cfield_moj_thumbnail_image%2Cfield_image%2Ctitle%2Cfield_moj_description%2Cfield_moj_series&fields%5Bnode--landing_page%5D=drupal_internal__nid%2Cfield_moj_thumbnail_image%2Cfield_image%2Ctitle%2Cfield_moj_description%2Cfield_moj_series&fields%5Bfile--file%5D=drupal_internal__fid%2Cid%2Curi',
```

Comment:
This feels a bit of a copout test? Although equally testing `addFields` calls feels a bit like testing the code not the functionality? Meh, I guess it shows query hasn't changed 🤷‍♀️

Reply:
as discussed!